intern/service/boss/v1/volume.py

```diff
@@ -277,6 +277,7 @@ def get_cutout(
                         # TODO: kwargs
                     )
                  for b in blocks])
+                pool.close()
```

Comment:
Do you mind making a minor change so we use a context for `pool` on line 271, instead?
```python
with multiprocessing.Pool(processes=parallel) as pool:
```

Reply:
That is good！I will do that.